### PR TITLE
Add Contributing Guideline

### PR DESCRIPTION
**Pull request by Huxpro**
As a follow-up of <https://github.com/facebook/jsx/pull/132#issuecomment-1050449593>, we should clarify the requirements and expectations on different kinds of contributions.

### CODE REVIEW

**Comment by Huxpro** [comment]
Feel free to comment or open an issue if you have any feedback

File: README.md [hunk 1]
```diff
@@ -104,6 +104,28 @@ Prior Art
 
 The JSX syntax is similar to the [E4X Specification (ECMA-357)](http://www.ecma-international.org/publications/files/ECMA-ST-WITHDRAWN/Ecma-357.pdf). E4X is a deprecated specification with deep reaching semantic meaning. JSX partially overlaps with a tiny subset of the E4X syntax. However, JSX has no relation to the E4X specification.
 
+
+Contributing
```
Comment (Josh-Cena) [inline README.md:108]: Move to a separate file `CONTRIBUTING.md` so we can take advantage of GitHub's UI? https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors

Comment (Huxpro) [reply README.md:108]: Make sense. Thanks for the feedback. Will do!

Comment (Josh-Cena) [reply README.md:108]: Just realized there's already a pending https://github.com/facebook/jsx/pull/114 😄

Comment (Huxpro) [reply README.md:108]: Hah yea I've been thinking of merge them into one ;)

**Pull request merged**